src/components/Menu/Menu.tsx

```diff
@@ -0,0 +1,89 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React, { PropsWithChildren, ReactElement, RefObject, useEffect, useState } from 'react';
+import { usePopper } from 'react-popper';
+import { merge } from '@utilities/merge';
+import { MenuItemProps } from '@components/MenuItem';
+import { useMenuKeyboardNavigation } from '@components/Menu/useMenuKeyboardNavigation';
+interface Props {
+    triggerRef?: RefObject<Element>;
+    open?: boolean;
+    children?: ReactElement<MenuItemProps> | ReactElement<MenuItemProps>[];
+    onClose?: () => void;
+}
+
+export type MenuProps = PropsWithChildren<Props>;
+
+const CONTAINER_BASE_CLASSES = 'tw-relative tw-bg-base tw-rounded tw-py-2 tw-shadow-mid tw-w-max';
```

Comment:
To me this indicates once again, that we need to have a reusable building blocks generic component, in this case, container.

Reply:
I agree with you. This is a specific one with the shadow and an inset shadow for the border, which I had to use a pseudo element to make it possible. But I totally get your point.
It would be great have some basic reusable elements.